bookkeeper-server/conf/bk_server.conf

```diff
@@ -319,49 +378,95 @@ zkEnableSecurity=false
 # co-ordination service(eg: zookeeper service).
 # When false, bookie will use its ipaddress for the registration.
 # Defaults to false.
-#useHostNameAsBookieID=false
+# useHostNameAsBookieID=false
+
+# Whether conf the bookie to listen for BookKeeper clients executed on the local JVM.
+# enableLocalTransport=false
+
+# Whether conf the bookie to disable bind on network interfaces,
+# this bookie will be available only to BookKeeper clients executed on the local JVM.
+# disableServerSocketBind=false
+
+# Whether sorted-ledger storage enabled (default true)
+# sortedLedgerStorageEnabled=ture
+
+# The skip list data size limitation (default 64MB) in EntryMemTable
+# skipListSizeLimit=67108864L
+
+# The number of bytes we should use as chunk allocation for
+# org.apache.bookkeeper.bookie.SkipListArena
+# skipListArenaChunkSize=4194304
+
+# The max size we should allocate from the skiplist arena. Allocations
+# larger than this should be allocated directly by the VM to avoid fragmentation.
+# skipListArenaMaxAllocSize=131072
 
 # Auto-replication
 # The grace period, in seconds, that the replication worker waits before fencing and
 # replicating a ledger fragment that's still being written to upon bookie failure.
-#openLedgerRereplicationGracePeriod=30
+# openLedgerRereplicationGracePeriod=30
 
 # Whether statistics are enabled
-#enableStatistics=true
+# enableStatistics=true
 
 # Stats Provider Class (if statistics are enabled)
 #statsProviderClass=org.apache.bookkeeper.stats.CodahaleMetricsProvider
 
-# SSL Provider (JDK or OpenSSL)
+## SSL configure
```

Comment:
I think all the `ssl` settings has been renamed to `tls`. we need make sure they are consistent here.